superscore/widgets/views.py

```diff
@@ -1447,7 +1449,7 @@ def create_context_menu(self, entry: Entry) -> QtWidgets.QMenu:
         """
         menu = QtWidgets.QMenu(self)
         if self.context_menu_options[MenuOption.OPEN_PAGE] and self.open_page_slot:
```

Comment:
```suggestion
        if self.context_menu_options[MenuOption.OPEN_PAGE]:
```
Same as above thread